scripts/core/index.ts

```diff
@@ -86,10 +87,29 @@ let _superdesk;
 
 core.component('sdExtensionPage', reactToAngular1(ExtensionPage, [], [], styles));
 core.config(['$routeProvider', 'superdeskProvider', ($routeProvider, superdesk) => {
+    // set initial default route to personal
+    // when user is logged in, it will be overwritten by a default route
+    // from configs if user has permissions to that route
     $routeProvider.when('/', {
-        redirectTo: appConfig.defaultRoute,
+        redirectTo: '/workspace/personal',
```

Comment:
shouldn't this be appConfig.defaultRoute so you don't have to redirect again below?

Reply:
That's what I tried first. I can't do this synchronously since I first need to check permissions for `defaultRoute` which is async. When it's not registered synchronously the login page breaks.